### DISCUSSION TITLE
Register Bootstrap 5 in Vue project

### DISCUSSION THREAD

**Discussion by ilarioni** [twbs/bootstrap#35550] [Q&A]
How can I register Bootstrap 5 in my Vue project (main.js) file properly?

is the following lines correctly? 
import 'popper.js'
import 'bootstrap'
import 'bootstrap/dist/css/bootstrap.min.css'

Bootstrap-vue documentation was perfect. In Bootstrap 5 documentation I failed to find this information.

**Comment by aqeelabpro** [comment]
*first run this in your project*
npm install bootstrap

_*Import this in main.js*_

**import BootstrapVue from 'bootstrap-vue'**

**Comment by mariusa** [comment]
https://getbootstrap.com/docs/4.0/getting-started/webpack/
